Jetchat/gradle/libs.versions.toml

```diff
@@ -9,10 +9,7 @@ androidx-activity-compose = "1.9.0"
 androidx-appcompat = "1.7.0"
 androidx-benchmark = "1.2.4"
 androidx-benchmark-junit4 = "1.2.4"
-androidx-compose-bom = "2024.06.00"
```

Comment:
If you rebase, you should be able to pick this up from main

Reply:
That's what I tried at first, but then it was giving me errors about the updated AGP version.... which should have just worked without a problem, so I'm not sure if it was just a local caching issue.